features/feature1/lib/src/presentation/pages/feature1_page.dart

```diff
@@ -28,11 +28,14 @@ class Feature1BodyPage extends StatelessWidget {
   Widget build(BuildContext context) {
     return BlocBuilder<Feature1Bloc, Feature1State>(
       builder: (context, state) {
+        LoggerUtil().info("current state: $state");
+
```

Comment:
After previous changes here we could access the logger doing something like this:
```suggestion
 InjectionModule.injector<AbstractLogger>().info("current state: $state");

```